scopes/defender/formatter/format.cmd.ts

```diff
@@ -43,7 +43,7 @@ export class FormatCmd implements Command {
   group = 'development';
   helpUrl = 'reference/formatting/formatter-overview';
   options = [
-    ['c', 'changed', 'format only new and modified components'],
+    ['c', 'changed', 'format only new and modified components'], // @davidF maybe we should align this with all other commands and do this by default (and add the unmodified flag)
```

Comment:
@GiladShoham is the owner of this command. To me it makes sense.

Reply:
I'm actually thinking about this again and maybe formatting all components makes sense by default, especially when run for the first time. And then have a flag if you dont want to be constantly re-formatting all components